### ISSUE TITLE
Merge Hackthetalks to new section in curriculum

### ISSUE THREAD

**Issue by iLienstAr** [HackYourFutureBelgium/hack-the-talks#1]
@colevandersWands can we move the #hack-the-talks readme to our Curriculum section?

Let's make a 'Career Skills Sessions'-section.

"Career Skills are the abilities you have to that enable you to do your job and to manage your career. These are over and above the skills and technical knowledge you need to perform the tasks that are part of your job."

**Comment by colevandersWands** [comment]
totally @iLienstAr .  we'll have to copy-paste the slides, I'm not sure how to do that through the Gitbook UI so I can do the first steps locally.  from there we can do wording and formatting from the gitbook.io

**Comment by iLienstAr** [comment]
@colevandersWands yes please. Let me know when it's copy pasted and when I can work on it from the Gitbook 🙏
